src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
```

Comment:
use either text values (regular, bold, etc) or number values (400, 500, 800) in your CSS, but not both together